src/libstd/io/buffered.rs

```diff
@@ -525,6 +525,25 @@ impl<W: Write> BufWriter<W> {
     #[stable(feature = "rust1", since = "1.0.0")]
     pub fn get_mut(&mut self) -> &mut W { self.inner.as_mut().unwrap() }
 
+    /// Returns a reference to the internally buffered data.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// # #![feature(bufreader_buffer)]
+    /// use std::io::BufWriter;
+    /// use std::net::TcpStream;
+    ///
+    /// let mut buf_writer = BufWriter::new(TcpStream::connect("127.0.0.1:34254").unwrap());
```

Comment:
This `mut` is unneeded for this example, right?

Reply:
I think most other examples return a result to avoid the `unwrap`:

```rust
fn main() -> std::io::Result<()> {
    let buf_writer = BufWriter::new(TcpStream::connect("127.0.0.1:34254")?);
```

Check to see what the prevailing pattern is and follow that, please.